crates/polars-lazy/src/dsl/list.rs

```diff
@@ -23,7 +23,7 @@ impl IntoListNameSpace for ListNameSpace {
 fn offsets_to_groups(offsets: &[i64]) -> Option<GroupsProxy> {
     let mut start = offsets[0];
     let end = *offsets.last().unwrap();
-    let fits_into_idx = (end - start) <= IdxSize::MAX as i64;
```

Comment:
Did you reach this limit? :astonished:

Reply:
I don't have 2**32 rows but I have 100M rows with ~1000 list elements each, which triggers an error in non-bigidx builds... I assume because IdxSize is used for list offsets, and I do have more than 2**32 list elements.

I'm also realizing that my fix is dumb because the check is always true. I'll defer to you on the right way to fix it :)